CMakeLists.txt

```diff
@@ -199,20 +207,28 @@ PKG_CONFIG_APPEND_LIBS(roboptim-core)
 # Search for dependencies.
 SET(BOOST_COMPONENTS date_time filesystem system unit_test_framework)
 SEARCH_FOR_BOOST()
-ADD_REQUIRED_DEPENDENCY("eigen3 >= 3.2.0")
-ADD_REQUIRED_DEPENDENCY("liblog4cxx >= 0.10.0")
+SEARCH_FOR_EIGEN("eigen3 >= 3.2.0")
```

Comment:
Good catch!

`roboptim-core.pc` before:
```
Requires: eigen3 >= 3.2.0, liblog4cxx >= 0.10.0
```

After:
```txt
Requires: liblog4cxx >= 0.10.0
```

So we need to add `eigen3` to `PKG_CONFIG_REQUIRES` if the pkg-config check was successful, or use `PKG_CONFIG_APPEND_CFLAGS` to add the include directory if the `FIND_PATCH` search was used.